recipes/libselinux/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class LibSELinuxConan(ConanFile):
+    name = "libselinux"
+    description = "Security-enhanced Linux is a patch of the Linux kernel and a number of utilities with enhanced security functionality designed to add mandatory access controls to Linux"
+    topics = ("conan", "selinux", "security-enhanced linux")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/SELinuxProject/selinux"
+    license = "Unlicense"  # This library (libselinux) is public domain software, i.e. not copyrighted
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    requires = ("pcre2/10.33",)
+
+    def _get_subfolders(self):
+        _sepol_subfolder = "libsepol-%s" % self.version
+        _selinux_subfolder = "libselinux-%s" % self.version
+        return _sepol_subfolder, _selinux_subfolder
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Only Linux is supported")
+
+    def build_requirements(self):
+        if not tools.which("flex"):
+            self.build_requires("flex/2.6.4")
```

Comment:
```suggestion
        self.build_requires("flex/2.6.4")
```

I prefer avoiding system apps. Why? If any problem occur, I won't be able to reproduce the user's scenario.